src/containerapp/azext_containerapp/containerapp_functions_decorator.py

```diff
@@ -267,15 +271,24 @@ def get_summary(self):
             # Fetch the app insights resource app id
             app_id = self._get_app_insights_id(resource_group_name, container_app_name, revision_name)
 
-            # Use application insights query to get function invocations summary
-            invocation_summary_query = (
-                f"requests | extend functionNameFromCustomDimension = tostring(customDimensions['faas.name']) "
-                f"| where timestamp >= ago({timespan}) "
-                f"| where cloud_RoleName =~ '{container_app_name}' "
-                f"| where cloud_RoleInstance contains '{revision_name}' "
-                f"| where operation_Name =~ '{function_name}' or functionNameFromCustomDimension =~ '{function_name}' "
-                f"| summarize SuccessCount = coalesce(countif(success == true), 0), ErrorCount = coalesce(countif(success == false), 0)"
-            )
+            # Build query based on active revision mode
```

Comment:
Do we still ask for revision parameter from user in single revision mode? If we are handling single revision mode separately then revision shouldn't be a compulsory param and only required in multi revision mode